code/Forms/UsedOnTable.php

```diff
@@ -72,6 +72,10 @@ public function usage(HTTPRequest $request)
         $record = $this->getRecord();
         $usage = $record->findOwners();
 
+        $usage = $usage->filterByCallback(function($item) {
```

Comment:
This is nasty and creates a coupling between campaigns and the used on table where there doesn't need to be one

Please remove this block.